website/README.md

```diff
@@ -5,25 +5,33 @@ This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern
 ## Installation
 
 ```console
-yarn install
+npm install
 ```
 
 ## Local Development
 
 ```console
-yarn start
+npm start
 ```
 
 This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
 
 ## Build
 
 ```console
-yarn build
+npm run build
 ```
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
+## Versioning
+
+```console
+npm run docusaurus docs:version <VERSION>
+```
+
+This command will tag a new version from master docs.
```

Comment:
Can you also mention the version sidebar and version directory path that is where it will create:
https://v2.docusaurus.io/docs/versioning/#tagging-a-new-version

Reply:
Yes, Sure.. Updating